tests/integration/azure/test_locations.py

```diff
@@ -252,3 +255,75 @@ def test_run_validate_acl(make_cluster_permissions, ws, make_user, make_cluster,
             env_or_skip("TEST_A_LOCATION"),
             changes=[PermissionsChange(remove=remove_azure_permissions, principal=user.user_name)],
         )
+
+
+def delete_ucx_created_resources(api_client):
+    """Delete UCX created resources"""
+    for resource in api_client.list():
+        if resource.name is not None and resource.comment is not None and resource.comment == "Created by UCX":
+            api_client.delete(resource.name, force=True)
+
+
+@pytest.fixture
+def clean_storage_credentials(az_cli_ctx):
+    """Clean test generated storage credentials."""
+    delete_ucx_created_resources(az_cli_ctx.workspace_client.storage_credentials)
+    yield
+    delete_ucx_created_resources(az_cli_ctx.workspace_client.storage_credentials)
+
+
+@pytest.fixture
+def clean_external_locations(az_cli_ctx):
+    """Clean test generated external locations."""
+    delete_ucx_created_resources(az_cli_ctx.workspace_client.external_locations)
+    yield
+    delete_ucx_created_resources(az_cli_ctx.workspace_client.external_locations)
+
+
+def test_run_external_locations_using_access_connector(
```

Comment:
This test fails:
1. Probably because the (non-external) access connector does not have rights on the mount.
2. External locations already exist for the mount